Cargo.toml

```diff
@@ -41,3 +41,6 @@ version = "=0.3.2"
 [dependencies.secp256k1]
 version = "0.11"
 features = [ "rand" ]
+
+[dev-dependencies]
+bincode = "1.0.1"
```

Comment:
Can you make this optional and depend on the `serde-decimal` feature?

I actually don't know if you can with `dev-dependencies`, and it's not a big deal because the actual `extern crate` code is feature-gated, but it'd be ideal if this weren't even built for people who don't want serde.